### PR TITLE
Update emoji-list.jsx

### PR DESCRIPTION
**Pull request by lfbrock**
Update help text based on feedback from IT, and add missing en.json line

### CODE REVIEW

**Comment by hmhealey** [comment]
+1

**Comment by jwilander** [comment]
+1

File: webapp/components/emoji/components/emoji_list.jsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ export default class EmojiList extends React.Component {
                 <span className='backstage-list__help'>
                     <FormattedMessage
                         id='emoji_list.help'
-                        defaultMessage='Custom emoji are available to everyone on your server and will show up in the emoji autocomplete menu.'
+                        defaultMessage='Custom emoji are available to everyone on your server and will show up in the emoji autocomplete menu. Other users may need to refresh the page before new emojis show up.'
```
Comment (it33) [inline webapp/components/emoji/components/emoji_list.jsx]: Super minor nit, maybe "show up" > "appear" in both instances above to make it easier to read for users for whom English isn't a first language?

File: webapp/i18n/en.json [hunk 2]
```diff
@@ -997,6 +997,7 @@
   "emoji_list.creator": "Creator",
   "emoji_list.delete": "Delete",
   "emoji_list.empty": "No Custom Emoji Found",
+  "emoji_list.help": "Custom emoji are available to everyone on your server and will show up in the emoji autocomplete menu. Other users may need to refresh the page before new emojis show up.",
```
Comment (it33) [inline webapp/i18n/en.json]: Super minor nit, maybe "show up" > "appear" in both instances above to make it easier to read for users for whom English isn't a first language?

**Pull request merged**